docs/Using Fleet/enroll-chromebooks.md

```diff
@@ -24,6 +24,11 @@ By default, the hostname for a Chromebook host will be blank. The hostname can b
 ## Debugging ChromeOS
 To learn how to debug the Fleetd Chrome extension, visit [here](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Testing-and-local-development.md#fleetd-chrome-extension).
 
+## Performance testing
+In August 2023, we completed scale testing on 10k Windows hosts and 70k macOS hosts. Ultimately, we validated both server and host performance at that scale.
```

Comment:
I'd consider changing this to just say "80k hosts" to avoid confusing people by talking about performance on other OS's in the Chromebook section of the docs (it confused me at first 😅)
```suggestion
In August 2023, we completed scale testing on 80k hosts. Ultimately, we validated both server and host performance at that scale.
```
Would also consider being up-front about the fact that we didn't actually test on Chromebooks, but that we don't anticipate significant differences in performance compared to the OS's we _did_ test.